include/wil/resource.h

```diff
@@ -2038,20 +2050,6 @@ namespace details
             }
         }
 
-        shared_storage(const shared_storage& other) WI_NOEXCEPT : m_ptr(other.m_ptr)

```

Comment:
I'm curious as to why some of these assignment/copy operations are being deleted.